trace/trace.go

```diff
@@ -98,24 +102,23 @@ func WithSpan(parent context.Context, s *Span) context.Context {
 
 // StartOptions contains options concerning how a span is started.
 type StartOptions struct {
-	// RecordEvents indicates whether to record data for this span, and include
-	// the span in a local span store.
-	// Events will also be recorded if the span will be exported.
+	// Deprecated: this option has no effect.
 	RecordEvents bool
```

Comment:
We still can remove the deprecated options.

Reply:
ok